graphql/types/paging.graphql

```diff
@@ -0,0 +1,8 @@
+type Paging {
+  page: Int!
+  perPage: Int!
+  total: Int!
+  pages: Int!
+  _from: Int!

```

Comment:
No need for this information because it can be derived. `page * perPage`

Reply:
yes we can remove it, but for the last page `to` value seems to be needed due to 
`number of items in last page <= page size`

Reply:
You can always do just : `page*perPage + list.length` to get this info

Reply:
it would be better to keep `total`, `page` and `perPage` as pagination result. that would solve problem of destroying the state.